axiosUtility.js

```diff
@@ -58,14 +58,14 @@ const httpRequest = async (options) => {
         error.response.data,
         options,
         error.response
-      );
+      )
     } else if (error.request) {
-      throw new RequestError(`No response received: ${error.message}`, options);
+      throw new RequestError(`No response received: ${error.message}`, options)
     } else {
-      throw new RequestError(`Request setup error: ${error.message}`, options);
+      throw new RequestError(`Request setup error: ${error.message}`, options)
```

Comment:
I'm in favor of adding an Eslint configuration file to enforce shared standards about details like semicolons vs no semicolons.

For me, the most important thing about such details is the consistency throughout the code base. 

That avoids people writing different sections in different styles and avoidable changes during peer-review.

I have personally trended towards the `airbnb` style but am open to whatever you prefer.

Reply:
It looks like right now, we run ESLint using the Standard config as part of our `yarn test` script (that's where I saw this and why I removed these). I also usually prefer Airbnb's style, I'm definitely open to switching to Airbnb for consistency. We can do it in this PR if you’d like, or handle it separately so we don’t mix style changes with functional ones. Let me know which approach you prefer!

Reply:
I agree that's better to not mix the the logic and style changes, so let's defer that.